pow-migration/src/monitor.rs

```diff
@@ -86,11 +124,27 @@ fn is_valid_ready_txn(
     genesis_config_hash: &String,
 ) -> bool {
     // Check if the txn contains extra data and matches our genesis config hash
-    if let Some(txn_data) = &txn.data {
-        if txn_data != genesis_config_hash {
-            return false;
-        }
-    } else {
+    let Some(txn_data) = &txn.data else {
+        return false;
+    };
+
+    if txn_data != genesis_config_hash {
+        return false;
+    }
+
+    // Check for the other readiness criteria
+    block_window.contains(&txn.block_number)
+        && txn.to_address == Address::burn_address().to_user_friendly_address()
+        && txn.value == 1
+}
+
+/// Checks if the provided transaction meets the criteria in order to be
+/// considered a valid online-transaction
+fn is_valid_online_txn(txn: &TransactionDetails, block_window: &Range<u32>) -> bool {
+    let Some(txn_data) = &txn.data else {
+        return false;
+    };
+    if txn_data != "online" {
         return false;
     }
```

Comment:
There is probably some variation of
```suggestion
   Some("online".to_string()) == txn.data &&
```
Also the value = 1 below here (not in changeset should not be a validity criteria.